tests/unit/test_column_noise.py

```diff
@@ -173,9 +178,23 @@ def test_generate_within_household_copies():
     pass
 
 
-@pytest.mark.skip(reason="TODO")
-def test_swap_months_and_days():
-    pass
+def test_swap_months_and_days(dummy_dataset):
```

Comment:
We're only testing noising dates in the format `MM/DD/YYYY`. We need to test that we're correctly noising `YYYY-MM-DD` dates as well.

Reply:
`YYYYMMDD` no hyphens, you mean.